src/redis_string.cc

```diff
@@ -54,6 +54,9 @@ rocksdb::Status String::getRawValue(const std::string &ns_key, std::string *raw_
     return rocksdb::Status::NotFound(kErrMsgKeyExpired);
   }
   if (metadata.Type() != kRedisString && metadata.size > 0) {
+    if (metadata.Type() == kRedisBitmap) {
```

Comment:
There's a bit tricky here to use the NotSupported error as implicitly the metadata type was bitmap. But it's a bit ugly if we also return type on this function, do you have any idea? @ShooterIT @ColinChamber

Reply:
For now, I actually don't have any perfect idea as well.

Reply:
It's fine to me before we have a better solution, @ShooterIT can take a look.

Reply:
To be honest, i don't like this design, there is my idea

- don't have any change here
- if returning an `InvalidArgument` error when getting from string db, then try to get from bitmap db. I think we don't usually operate against wrong types in production environments, so don't worry performance loss

Reply:
Thanks.I will modify it according to your suggestion.